demo/monitoring_all_process_instances/index.html

```diff
@@ -0,0 +1,94 @@
+            /* width is relative to the window size */
+            height: 650px;
+            border-style: solid;
+            border-color: #B0B0B0;
+            border-width: 1px;
+            /* This ensures that the parts of the diagram outside of the container are not displayed. */
+            overflow: hidden;
+            background-color: white;
+        }
+    </style>
+</head>
+<body>
+<header class="navbar bg-dark p-2">
+    <section class="navbar-section ml-2">
+        <img src="../../examples/static/img/logo_64x64.ico" alt="logo" class="logo">
+        <span class="h3 text-light ml-2 lib-name-text">BPMN Visualization Examples</span>
+    </section>
+    <section class="navbar-section">
+        <a href="https://process-analytics.github.io/bpmn-visualization-js/" class="mr-2 btn btn-secondary">Docs</a>
+        <div class="dropdown mr-2">
+            <a href="#" class="btn btn-primary dropdown-toggle" tabindex="0">
+                GitHub <span class="icon icon-caret icon-in-the-middle"></span>
+            </a>
+            <ul class="menu">
+                <li><a href="https://github.com/process-analytics/bpmn-visualization-js"
+                       class="mr-2 btn btn-link">Library</a></li>
+                <li><a href="https://github.com/process-analytics/bpmn-visualization-examples"
+                       class="mr-2 btn btn-link">Examples</a></li>
+            </ul>
+        </div>
+    </section>
+</header>
+
+<section class="container col-10" style="margin-top: 3rem">
+    <div class="col-11 col-mx-auto">
+        <div class="h2">Monitoring of all process instances demo</div>
+
+        <div class="col-12 pt-2">
+            <div id="controls" class="col-12" style="margin-top: 1rem; margin-bottom: 2rem">
+                <div id="switch-panel" class="switch switch--horizontal switch--expanding-inner">
+                    <input id="btn-time" type="radio" name="switch-data-type" checked="checked"/>
+                    <label for="btn-time">Time</label>
+                    <input id="btn-frequency" type="radio" name="switch-data-type"/>
+                    <label for="btn-frequency">Frequency</label>
+                    <span class="toggle-outside">
+                        <span class="toggle-inside"></span>
+                    </span>
+                </div>
+            </div>
+            <div class="bg-dark col-12 p-1">
+                <div class=" m-1">
+                    <div id="time-bpmn-container" class="col 12 bpmn-container"></div>
+                    <div id="frequency-bpmn-container" class="col-12 bpmn-container d-hide"></div>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+
+<script src="../../examples/static/js/link-to-sources.js"></script>
+<!-- load bpmn-visualization -->
+<script src="https://cdn.jsdelivr.net/npm/bpmn-visualization@0.14.1/dist/bpmn-visualization.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/values.js"></script>
```

Comment:
⚠️ I suggest we use a specific version to ensure no breaking changes. This will also allow browser caching and setting the [SRI](https://www.jsdelivr.com/using-sri-with-dynamic-files)